task/06-conditions-n-loops-tasks.js

```diff
@@ -63,7 +71,12 @@ function getFactorial(n) {
  *   -1,1  =>  0  ( = -1 + 0 + 1 )
  */
 function getSumBetweenNumbers(n1, n2) {
-    throw new Error('Not implemented');
+    let sum = 0;
+    for( let i = n1; i<=n2; i ++)
```

Comment:
Need to use indents between operators.
It is necessary to correctly indent the brackets.
https://javascript.info/coding-style

Reply:
Fixed.